packages/samples/mvp-dapp/src/views/load.view.ts

```diff
@@ -0,0 +1,60 @@
+// import type { StoreType } from '../record/record.model'
+
```

Comment:
```suggestion
```

Reply:
Will be uncommented after model branch merged.